plugins/woocommerce/src/Admin/RemoteInboxNotifications/SpecRunner.php

```diff
@@ -101,7 +101,10 @@ private static function get_url( $action ) {
 		}
 
 		if ( isset( $action->url_is_admin_query ) && $action->url_is_admin_query ) {
-			return wc_admin_url( $action->url );
+			if ( strpos( $action->url, '&path' ) === 0 ) {
+				return wc_admin_url( $action->url );
```

Comment:
I decided to go this route, as `&path` seemed to be the only examples I could find of when we were linking to WCA pages.
As the `url_is_admin_query` entails, we likely want to phase out that support and always provide a WP Admin link even for WCA pages, but that is just a thought.

Reply:
This seems like it will catch most known cases.  Just for the sake of discussion, is there any chance we can reuse @joelclimbsthings's PR ( https://github.com/woocommerce/woocommerce/pull/33124/files ) to detect and navigate to links in the client?

>As the url_is_admin_query entails, we likely want to phase out that support and always provide a WP Admin link even for WCA pages, but that is just a thought.

Completely agree with this.  It probably would have been wise in retrospect to never allow this specific level of partial URLs only for WCA.

Reply:
> Just for the sake of discussion, is there any chance we can reuse @joelclimbsthings's PR ( https://github.com/woocommerce/woocommerce/pull/33124/files ) to detect and navigate to links in the client?

We could use his `navigateTo` function in the inbox-note, but that would still require the above logic. As the current logic in Joel's PR can't differentiate between going to `&path=wc-addons` or `plugins.php`.
So it would only work well for going to either WCA pages or non wp admin pages.

The one benefit of using Joel's `navigateTo` in the inbox-note is that if we were going to a WCA page, it won't do a full page refresh, but makes use of the react router (I think) not a 100% on this.

Reply:
Ya, that makes sense.  This was definitely an oversight on our part allowing partial (WCA only) paths as options.

I may p2 this later to make sure we're all on the same page going forward.